kpconv_torch/datasets/common.py

```diff
@@ -1,10 +1,19 @@
+"""
+Mother Dataset class
+
+@author: Hugues THOMAS, Oslandia
+@date: july 2024
+"""
+
+# pylint: disable=R0913, R0914, R0912, R0902, R0915, E0401, C0103
```

Comment:
to allow pre-commit to finish

Reply:
Why don't you use the same mark as in the CI?

Reply:
I did that, but neither the CI, nor pre-commit were ok, since we had to lower the threshold (temporarily, as you said), that is why I then tried to improve the code, but to avoid to spend too much time on it, I left on the side the errors / warnings of the linter that seemed the less important to me, to be able to focus on the others. 

Finally, if you want, we can remove them all and see if the threshold is ok.

I am sorry, but I do not know how to manage such a situation, where the linter is blocking because of the initial quality.

Reply:
Afterwards, I removed the exceptions and the code was valued as more than 8 on the squashed commits where the linter was applied. I just invalidated the `invalid-name` error which corresponds apparently to an error about modules import, that I can't understand.